interface/patient_tracker/patient_tracker_status.php

```diff
@@ -116,8 +116,8 @@
             </div>
             <div class="position-override">
                 <div class="btn-group oe-opt-btn-group-pinch" role="group">
-                    <a href='javascript:;' class='btn btn-secondary btn-save' onclick='document.getElementById("form_note").submit();'><?php echo xlt('Save')?></a>
-                    <a href='javascript:;' class='btn btn-link btn-cancel oe-opt-btn-separate-left' onclick="dlgclose();" ><?php echo xlt('Cancel'); ?></a>
+                    <button type="button" class='btn btn-primary btn-save btn-sm' onclick='document.getElementById("form_note").submit();'><?php echo xlt('Save')?></button>
+                    <button type="button" class='btn btn-secondary btn-cancel btn-sm oe-opt-btn-separate-left' onclick="dlgclose();" ><?php echo xlt('Cancel'); ?></button>
```

Comment:
This PR looks good and bringing in.

Unrelated issue, though:
@stu01509 and @tywrenn , does it make sense to completely remove the oe-opt-btn-separate-left class since not used in many places and was part of a previous UI effort that appears to no longer be used.

Reply:
OK, I will remove it 👍